test/shortcuts.test.js

```diff
@@ -155,19 +155,19 @@ test('[shortcuts] lambda', (assert) => {
       TargetArn: 'arn:aws:sqs:us-east-1:123456789012:queue/fake'
     },
     Description: 'my description',
-    Environment: { Variables: { A: 'a' } },
+    Environment: { Variables: { MyCoolEnv: 'a' } },
     FunctionName: 'my-function',
     Handler: 'index.something',
     KmsKeyArn: 'arn:aws:kms:us-east-1:123456789012:key/fake',
-    Layers: ['arn:aws:fake:layer/abc'],
+    Layers: ['arn:aws:lambda:us-east-2:590474943231:layer:AWS-Parameters-and-Secrets-Lambda-Extension:4'],
     MemorySize: 512,
     ReservedConcurrentExecutions: 10,
-    Runtime: 'nodejs18.x',
+    Runtime: 'nodejs22.x',
     Tags: [{ Key: 'a', Value: 'b' }],
     Timeout: 30,
     TracingConfig: { Mode: 'Active' },
     VpcConfig: {
-      SecurityGroupIds: ['sg-12345'],
+      SecurityGroupIds: ['sg-12345678'],
```

Comment:
new cfn-lint is catching these as invalid security groups thus we shouldnt test with these